kotlintest-assertions/src/main/kotlin/io/kotlintest/eventually/Eventually.kt

```diff
@@ -1,5 +1,8 @@
-package io.kotlintest
+package io.kotlintest.eventually
```

Comment:
Moving this guy to another package breaks client's code. 

If it breaks already, we don't even need to deprecate, I suppose

Reply:
I'll add a deprecated alias.